test/jackdaw/serdes/avro_test.clj

```diff
@@ -483,5 +483,5 @@
     (is (= (String. (round-trip serde "bananas" (.getBytes "hello")))
            "hello"))
 
-    (is (thrown? org.apache.kafka.common.errors.SerializationException
```

Comment:
Do you think the fact that this has changed means we have no business including a test for it? Can you remember the original motivation?

Reply:
I really don't.